AppImage-package/bundle.sh

```diff
@@ -20,12 +20,17 @@ cd $HOME/$APP/
 wget -q https://github.com/probonopd/AppImages/raw/master/functions.sh -O ./functions.sh
 . ./functions.sh
 
+URLS=$(apt-get install -qq --reinstall --print-uris  python-tk | cut -d "'" -f 2)
+wget $URLS
```

Comment:
This does not make much sense, why the place of the apt-get calls makes any difference?